### PR TITLE
Adds new responder 

### PR DESCRIPTION
**Pull request by pwszpl**
Add a new responder which allow to see all variables that are defined in page tree.

### CODE REVIEW

File: test/fitnesse/responders/ScopeVariablesResponderTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    WikiPageUtil.addPage(childPage, PathParser.parse("GrandChildPage"), "simple content");
+  }
+
+  @Test
+  public void shouldListVariables() throws Exception {
+    request.setResource("SimplePage.ChildPage.GrandChildPage");
+    SimpleResponse response = (SimpleResponse) responder.makeResponse(context,request);
+    assertHasRegexp("<a href=\"SimplePage\">SimplePage</a>", response.getContent());
+    assertHasRegexp("<a href=\"SimplePage\\.ChildPage\">SimplePage.ChildPage</a>", response.getContent());
+  }
```
Comment (jediwhale) [inline test/fitnesse/responders/ScopeVariablesResponderTest.java:41]: Test that variable names and values are in the response?

**Review by jediwhale — CHANGES_REQUESTED** [review]
Thanks for this PR.

Please add a one line description in fitnesse/FitNesseRoot/FitNesse/UserGuide/AdministeringFitNesse/RestfulServices

File: test/fitnesse/responders/ScopeVariablesResponderTest.java (hunk 1 above)
Comment (pwszpl) [reply test/fitnesse/responders/ScopeVariablesResponderTest.java:42]: Good idea ;) I have extended test in last commit.

**Comment by pwszpl** [comment]
> Please add a one line description in fitnesse/FitNesseRoot/FitNesse/UserGuide/AdministeringFitNesse/RestfulServices

Added

**Pull request merged**